probcal/enums.py

```diff
@@ -11,7 +11,8 @@ class AcceleratorType(Enum):
 class HeadType(Enum):
     GAUSSIAN = "gaussian"
     POISSON = "poisson"
-    NEGATIVE_BINOMIAL = "negative_binomial"
+    NEGATIVE_BINOMIAL = "nbinom"
```

Comment:
Any specific reason we changed the value for `NEGATIVE_BINOMIAL`? This could break our support for older configs

Reply:
On some of the code that I was looking at I saw "nbinom" used, I can make sure that it gets switched back to "negative_binomial"

Reply:
Yeah, I think nbinom is used in some filenames, but the value for configs should be `negative_binomial`. Probably could be cleaned up in the future haha